symengine/tests/test_symbol.py

```diff
@@ -8,6 +8,10 @@ def test_symbol():
     assert str(x) == "x"
     assert str(x) != "y"
     assert repr(x) == str(x)
+    # Verify the successful use of slots.
+    assert hasattr(x, "__slots__")
```

Comment:
In Cythkn classes even `__slots__`does not exist

Reply:
Feel free to interrupt the builds.

Reply:
I'm abusing Travis a little because I don't feel like setting up the build environment right now.